src/bindings/js/node/lib/addon.ts

```diff
@@ -21,6 +21,12 @@ type elementTypeString =
   | 'f32'
   | 'string';
 
+type Any = string | number | boolean;
+
+type AnyMap = {
+  [propertyName: string]: Any;
+};
```

Comment:
```suggestion
type AnyMap = Record<string, *AnyNewName*>;
```
https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type

Reply:
Also, propose to replace `AnyMap` name

Reply:
Or just explicitly add `Record` type in the place, where it uses. 
Because our goal not to map values between C++ and JS sides, we should help developers by giving hints.
For `AnyMap` it looks like: 
![{EB9E1A7E-3DC9-4295-9DA8-AA946545726F}](https://github.com/user-attachments/assets/d92821a3-afcb-4efd-ba26-718f2a1efdca)

If specify inline, it will give clearer hint:
![{8E82A6E1-0CB1-40B9-BD71-A255B95FA5E7}](https://github.com/user-attachments/assets/af5e835d-9d6d-46da-ae5e-8d728ef8a360)

Reply:
I removed definition of `AnyMap`. Agree that built-in `Record` type is a simpler way to represent a map of string keys to OVAny values and provides better hints.